.github/workflows/ci.yml

```diff
@@ -185,6 +185,30 @@ jobs:
           yarn add -s prettier
           yarn exec -s prettier -- --check src/**/*.{ts,tsx}
 
+
+  python-formatting:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.x"
```

Comment:
Contact @iburakov about more specific version.

Reply:
Decided to try using 3.11